extensions/amp-youtube/1.0/base-element.js

```diff
@@ -23,12 +23,12 @@ BaseElement['Component'] = Youtube;
 
 /** @override */
 BaseElement['props'] = {
-  'autoplay': {attr: 'autoplay', type: 'boolean'},
-  'loop': {attr: 'loop', type: 'boolean'},
-  'controls': {attr: 'controls', type: 'boolean'},
-  'videoid': {attr: 'data-videoid'},
-  'liveChannelid': {attr: 'data-live-channelid'},
-  'dock': {attr: 'dock'},
-  'credentials': {attr: 'credentials'},
+  'autoplay': {attr: 'autoplay', type: 'boolean', media: true},
```

Comment:
Similar to video. I think only `dock` makes sense.

Reply:
Same as above, left `controls` but curious why you think it should be removed.